packages/interaction/src/properties/is-present.js

```diff
@@ -9,6 +9,11 @@ import { computed } from './helpers';
  */
 export default function(selector) {
   return computed(function() {
-    return !!this.$$(selector).length;
+    try {
+      // throws if the root element cannot be found
+      return !!this.$$(selector).length;
+    } catch (e) {
```

Comment:
Does eslint complain about an unused var?

Reply:
Not for `catch` because the `error` var is required

Reply:
TIL. I remember something like that..